src/app/styles/colors.styl

```diff
@@ -0,0 +1,2 @@
+
+$md-grey-300 = rgb(224, 224, 224)
```

Comment:
why some .styl and some .scss?

Reply:
So, when I started this project a year ago, I felt like trying out [stylus](http://stylus-lang.com/), which is another CSS pre-processor distinct from (but similar to) [Sass](http://sass-lang.com/). It's cool, I've been enjoying using it, but I don't particularly want to use two different forms of CSS processing in the project, so this is something worth resolving at some point, I think.

The reason I started using .scss is that MDC-web uses .scss files, and I wasn't sure if .styl files could import and interface with .scss files. So for the [src/index.scss](https://github.com/milotoor/xwordz/pull/2/files/cc519efd0eee870fb71c243007b5aeedbd3dadec#diff-5ab26e17c78a72ee4bc1774adbcf305e) file you commented on, I used .scss because it needed to directly interface with the MDC-web stylesheets (to set up the theme colors). It might be worth a small investigation to see if .styl files can interface with .scss mixins and shit (I don't think there's any fundamental reason they *shouldn't* be able to